buster/chatbot.py

```diff
@@ -78,25 +80,24 @@ def _init_documents(self):
         logger.info(f"embeddings loaded.")
 
     def _init_unk_embedding(self):
-        logger.info("Generating UNK token...")
```

Comment:
I don't like magic function in a constructor. I would suggest using a dataclass, with a factory method in the config where this function is called.

Reply:
I agree it's a little clunky, but this method will probably evolve over time, i'd rather not spend too much time factoring it right now as it'll likely become more important in the future